online_portfolio/static/portfolio/js/edit.js

```diff
@@ -708,7 +714,16 @@ function updateCategoryName(element, slideIndex) {
         data: {"oldName": oldName, "newName": newName},
         dataType: "json",
         success: function (data){
-            // add name change logic for frontend.
+            const headings =  document.querySelectorAll(".sectionHeadingDiv");
+            headings.forEach(heading=>{
+                const headingCategory = heading.getAttribute('category');
+                if(headingCategory === oldName){
+                    heading.setAttribute('category',newName);
+                    heading.querySelector("h2").innerHTML=newName;
```

Comment:
@HarshitaAgarwal09 this is fine, but I was thinking is there a better way? I mean if we in the future accidentally add another `<h2>` element in the `sectionHeadingDiv` this logic will break. It's not maintainable.

Reply:
Also, we still missed changing the category attribute in lines 542, 547, and 545. I have no idea how come I not mention those. Basically after deletion `oldName` should not exist in the frontend, and should be replaced by `newName` be it an attribute or value or text.

Reply:
I'd advise you to check that skill section has no trace of `oldName` once deleted.

Reply:
Okay

Reply:
@manthanchauhan  I am unable to access the category property from the tags as it is the custom attribute.

Reply:
@manthanchauhan Updated the code. Please check it

Reply:
@ there is no other way to acces the element h2 without accessing it this way. Also updated the code ,Please check.